rpc_jobs/rpc_upgrades.yml

```diff
@@ -42,14 +61,14 @@
         action: "kilo_to_newton_leap"
 
 - project:
-    name: "rpc-upgrades-aio-newton-release"
+    name: "rpc-upgrades-aio-newton-release-pm"
     repo_name: "rpc-upgrades"
     repo_url: "http://github.com/rcbops/rpc-upgrades"
     series: "master"
```

Comment:
Same as above, series/branches not needed for PM.

Reply:
Reason I did that is because then I collide on the two first projects since I split the first two out to avoid having to make a ton of excludes.